go/ai/gen.go

```diff
@@ -108,6 +108,7 @@ type ModelInfo struct {
 	Label    string             `json:"label,omitempty"`
 	Supports *ModelInfoSupports `json:"supports,omitempty"`
 	Versions []string           `json:"versions,omitempty"`
+	ConfigSchema map[string]any `json:"configSchema,omitempty"`
```

Comment:
This needs to come from `genkit-tools` otherwise it will get overwritten on next generation.